iceoryx_hoofs/source/relocatable_pointer/base_relative_pointer.cpp

```diff
@@ -21,13 +21,17 @@ namespace iox
 {
 namespace rp
 {
-BaseRelativePointer::BaseRelativePointer(ptr_t ptr, id_t id) noexcept
+// NOLINTJUSTIFICATION NewType is trivially copyable, hence pass by value is preferred
+// NOLINTNEXTLINE(performance-unnecessary-value-param)
+BaseRelativePointer::BaseRelativePointer(const ptr_t ptr, const id_t id) noexcept
     : m_id(id)
     , m_offset(computeOffset(ptr))
 {
 }
 
-BaseRelativePointer::BaseRelativePointer(offset_t offset, id_t id) noexcept
+// NOLINTJUSTIFICATION NewType is trivially copyable, hence pass by value is preferred
```

Comment:
It is again about size. Please change the justification. This occurs more often later in the file, but I will not mark it.